src/styles/shared/_buttons.scss

```diff
@@ -139,6 +140,7 @@
   &:active {
     border: border-width() solid var(--p-border);
     box-shadow: none;
+    color: var(--p-text);
```

Comment:
I think it can also be removed here

Reply:
Also note, below under the `pressed` state there's a `color: var(--p-button-text);` Which I don't think exists

Reply:
@dleroux the main cause of this error is overwriting/inheritance of CSS properties when two classes are used.. i.e. in  `<Button primary outline>Text</Button>` `.outline:hover` color is inherited from the `.primary:hover` class since it is not defined for `.outline:hover`. Fix add color to `.outlined:hover`.
Since color is also not defined in `:focus` and `:active` this will likely occur again for these if the class used along `.outline` has a color defined in these selectors. [Thats why I initially added `color` in these selectors too].

Didnt remove `var(--p-button-text);` because it is used in other parts of the `_button.scss` and is outside the scope of the bug. Maybe we create another bug for it to be removed?